Readme.md

```diff
@@ -1,21 +1,309 @@
+[PA]: https://pipal.in/
+[ipytail]: https://github.com/pipalacademy/ipytail
+
 # PipalHub - JupyterHub setup for [Pipal Academy][PA]
 
 PipalHub is JuputerHub setup optimized for remote workshops of Pipal Academy.
 
 It contains a JupyterHub server providing one Jupyter instance for each participant, a bunch of scripts to summarize changes and export notebooks as HTML so that instructor can quickly glance though the notebooks of the participants.
 
-## Implementation
+## Quick setup
 
-It is implemented using docker-compose with the following docker containers, and nginx.
+There are some easy-install scripts that can be used to setup the node with all dependencies
+that will be needed to run the server.
 
-* jupyterhub - for running jupyterhub server
-* build - script to [summarize the notebooks][ipytail] and export them as HTML
-* nginx - exposing the jupyterhub to the external world and handling SSL
+See the [section on adding users and packages](#adding-users-and-packages) after setup
+for how to add users or pip dependencies.
 
-[PA]: https://pipal.in/
-[ipytail]: https://github.com/pipalacademy/ipytail
+### Easy install
+
+#### `create-node.py` script (recommended)
+
+This one script can be used from your local machine to create a new node on DigitalOcean and
+get PipalHub up and running on it.
+
+**Description:**
+
+This script will create a DigitalOcean droplet with the given size and name, assign a DNS entry to it, and run the
+setup-node.sh script on it with the given hostname + configured base domain.
+
+Some of its default configuration can be found in the script after imports.
+[These](https://github.com/pipalacademy/pipalhub/blob/b10a5fa8817c4f3e594a0bd4f5044c157f5092da/create-node.py#L14-L21)
+can be changed if needed.
```

Comment:
Some defaults are hardcoded as constants at the [beginning of this file](https://github.com/pipalacademy/pipalhub/blob/b10a5fa8817c4f3e594a0bd4f5044c157f5092da/create-node.py#L14-L21).
These can be changed as needed.

```
Some defaults are hardcoded as constants at the [beginning of this file](https://github.com/pipalacademy/pipalhub/blob/b10a5fa8817c4f3e594a0bd4f5044c157f5092da/create-node.py#L14-L21).
These can be changed as needed.
```